pytensor/compile/function/types.py

```diff
@@ -1106,6 +1106,18 @@ def sync_shared(self):
         # NOTE: sync was needed on old gpu backend
         pass
 
+    def dprint(self, **kwargs):
+        """Debug print itself
+
+        Parameters
+        ----------
+        kwargs:
+            Optional keyword arguments to pass to debugprint function.
+        """
+        from pytensor.printing import debugprint
+
+        return debugprint(self, **kwargs)
+
```

Comment:
I have mixed feelings about this solution since this paradigm is quite rare, but perhaps it shouldn't be.

```suggestion
    from pytensor.printing import debugprint
```

I think this does everything cleanly, but I haven't tested it.

Reply:
Does this call the function on self?

Reply:
On class instances, yes.

Reply:
Thats :exploding_head: haha